carla_integration/VehicleConfigParams.yaml

```diff
@@ -1,16 +1,14 @@
-use_sim_time: true
```

Comment:
Is this parameter not needed? Does CARMA's ros Time value not come from CARLA in the current configuration?

Reply:
Added the use_sim_time parameter to config file which it was deleted accidentally